Turn OneAgent Enrichment on by default and update doc (#2)
dynatrace-oss-contrib/spring-boot: spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

	      dynatrace:
	        api-version: "v2"

> 	        api-token: "YOUR_TOKEN"

Reviewer: ```suggestion
	        api-token: "YOUR_TOKEN" # should be read from environment variables or properties
```

	        uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"

>             # or "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.

Reviewer: ```suggestion
	        # or "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
```
Otherwise it's not aligned when rendered (tabs vs spaces).

	          key2: "value2"
----

===== API v1 (Legacy)

The Dynatrace API v1 registry pushes metrics to the configured URI periodically using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v1/[Timeseries API v1].

> To export metrics to {micrometer-registry-docs}/dynatrace[Dynatrace], your API token, device ID, and URI must be provided:

Reviewer: ```suggestion
For backwards-compatibility with existing setups, `api-version` defaults to `"v1"` but can also be specified explicitly as such.
To export metrics to {micrometer-registry-docs}/dynatrace[Dynatrace], your API token, device ID, and URI must be provided:
```

> For the v1 API, the base URI needs to be specified, excluding the path.
> It will be added automatically.

Reviewer: ```suggestion
For the API v1, the base environment URI needs to be specified, without any path.
The API v1 endpoint path will be added automatically.
```

[source,yaml,indent=0,configprops,configblocks]
----
	management:
	  metrics:
	    export:
	      dynatrace:
	        api-token: "YOUR_TOKEN"
	        device-id: "YOUR_DEVICE_ID"
	        uri: "https://{your-environment-id}.live.dynatrace.com"

>             # or "https://{your-domain}/e/{your-environment-id}" on managed deployments.

Reviewer: ```suggestion
	        # or "https://{your-domain}/e/{your-environment-id}" on managed deployments.
```

	          key2: "value2"
----

===== API v1 (Legacy)

The Dynatrace API v1 registry pushes metrics to the configured URI periodically using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v1/[Timeseries API v1].
For backwards-compatibility with existing setups, `api-version` defaults to `"v1"` but can also be specified explicitly as such.
To export metrics to {micrometer-registry-docs}/dynatrace[Dynatrace], your API token, device ID, and URI must be provided:
For the API v1, the base environment URI needs to be specified, without any path.
The API v1 endpoint path will be added automatically.

[source,yaml,indent=0,configprops,configblocks]
----
	management:
	  metrics:
	    export:
	      dynatrace:

> 	        api-token: "YOUR_TOKEN"

Reviewer: ```suggestion
	        api-token: "YOUR_TOKEN" # should be read from environment variables or properties
```
